recipe/build.sh

```diff
@@ -14,6 +14,13 @@ else
       export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0;8.6"
   elif [[ ${cuda_compiler_version} == 11.2* ]]; then
       export TORCH_CUDA_ARCH_LIST="$TORCH_CUDA_ARCH_LIST;6.0;6.1;7.0;7.5;8.0;8.6"
+  elif [[ ${cuda_compiler_version} == 11.8 ]]; then
+      export TORCH_CUDA_ARCH_LIST="3.5;5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9"
```

Comment:
Was just noticing PTX isn't added. Looks like [PyTorch just adds this to latest]( https://github.com/conda-forge/pytorch-cpu-feedstock/blob/7c7a57b7515eaeda67d3879b56b68466f38f0b0d/recipe/build_pytorch.sh#L148 )

Would we want to do this here? If so, would we want to do this for other CUDA versions?

```suggestion
      export TORCH_CUDA_ARCH_LIST="3.5;5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9+PTX"
```